test/distributed/org/apache/cassandra/distributed/test/thresholds/TombstoneCountWarningTest.java

```diff
@@ -327,4 +361,75 @@ private static ResultSet driverQueryAll(String cql)
     {
         return JAVA_DRIVER_SESSION.execute(new SimpleStatement(cql).setConsistencyLevel(com.datastax.driver.core.ConsistencyLevel.ALL));
     }
+
+    @Shared
+    public static class State
+    {
+        public static volatile InetSocketAddress blockFor = null;
+        public static volatile CompletableFuture<Void> promise = new CompletableFuture<>();
```

Comment:
could be `null`... I just hate nulls ^_^

Reply:
I guess the benefit of it starting out `null` would be that, if it must be set in `blockFor()`, null would alert us immediately if we had failed to do that? (...or aesthetically, we return it to `null` in `syncAndClear()`, so why not here?)

Reply:
I went with null